src/utilities/computational_analysis.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jan 28 2023.
+
+@author: julie-vienne
+"""
+
+import numpy as np
+import os
+
+
+
+class ComputationalBurdenAnalyzer:
+    """
+    Performs and saves calculations on computational burden of
+    solving the optimiaztion or running a pandapower simulation
+
+    """
+
+    def __init__(self, prm):
+        """
+        Initialise ComputationalBurdenAnalyzer object.
+
+        inputs:
+        prm:
+            input parameters
+        """
+
+        # subset line losses
+        for info in [
+            'subset_line_losses_modelled'
+            ]:
+            setattr(self, info, prm['grd'][info])
+
+    def _save_computational_burden_opti(
```

Comment:
save things in `Record` object. This is where the recording of things is managed.

Reply:
I see, thanks for the advice. I think I'm having a hard time understanding how to add the results in my newly created entry in `Record`. For example, I have created and initialized `opti_timer` in `Record` and I want to populate it with the timer of the optimization. What is the best way to do this without initializing the class `Record` in the optimizer?

Reply:
you can look for example at how `duration_epoch` is saved in the `end_epoch` method in `record`.